src/nunit-gui/Model/ResultSummary.cs

```diff
@@ -0,0 +1,127 @@
+// 
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+// LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+// OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+// WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+// ***********************************************************************
+
+using System;
+
+namespace NUnit.Gui.Model
+{
+    /// <summary>
+    /// Summary description for ResultSummary.
+    /// </summary>
+    public class ResultSummary
+    {
+        /// <summary>
+        /// Returns the number of test cases actually run.
+        /// </summary>
+        public int RunCount => PassCount + FailureCount + ErrorCount + InconclusiveCount;
+
+        /// <summary>
+        /// Returns the number of test cases not run for any reason.
+        /// </summary>
+        public int NotRunCount => IgnoreCount + ExplicitCount + InvalidCount + SkipCount;
+
+        /// <summary>
+        /// Returns the number of failed test cases (including errors and invalid tests)
+        /// </summary>
+        public int FailedCount => FailureCount + InvalidCount + ErrorCount;
+
+        /// <summary>
+        /// Returns the sum of skipped test cases, including ignored and explicit tests
+        /// </summary>
+        public int TotalSkipCount => SkipCount + IgnoreCount + ExplicitCount;
+
+        public string OverallResult { get; set; }
+
+        public double Duration { get; set; }
+
+        public DateTime StartTime { get; set; }
+
+        public DateTime EndTime { get; set; }
+
+        /// <summary>
+        /// Gets the number of test cases for which results
+        /// have been summarized. Any tests excluded by use of
+        /// Category or Explicit attributes are not counted.
+        /// </summary>
+        public int TestCount { get; set; }
+
+        public int WarningCount { get; set; }
+
+        /// <summary>
+        /// Gets the count of passed tests
+        /// </summary>
+        public int PassCount { get; set; }
+
+        /// <summary>
+        /// Gets the count of failed tests, excluding errors and invalid tests
+        /// </summary>
+        public int FailureCount { get; set; }
+
+        /// <summary>
+        /// Returns the number of test cases that had an error.
+        /// </summary>
+        public int ErrorCount { get; set; }
+
+        /// <summary>
+        /// Gets the count of inconclusive tests
+        /// </summary>
+        public int InconclusiveCount { get; set; }
+
+        /// <summary>
+        /// Returns the number of test cases that were not runnable
+        /// due to errors in the signature of the class or method.
+        /// Such tests are also counted as Errors.
```

Comment:
I would think that "counted as Errors" => "counted as Failed". Otherwise we would count the testcase twice in `FailedCount`.